### DISCUSSION TITLE
i'm not sure i got exactly what posts can be seen

### DISCUSSION THREAD

**Discussion by lucajet** [snarfed/bridgy-fed#1023] [General]
Hello, 
right now i bridged two accounts

https://bsky.app/profile/underscorner.mastodon.uno.ap.brid.gy  (a mastodon instance)
https://bsky.app/profile/luca.sironi.tk.ap.brid.gy (a pleroma one, with an issue with the follow to the bot stucked)

One person with a bridged bluesky account, accepted my follow.
I was able to put a like to a post from that person, with both accounts.

Anyway i was expecting all my public posts after being bridged, to reach bluesky.
So far i see none.
Could be related to the fact that both accounts are setted to manually approve followers ?
Aside that, my posts are always public.

https://fed.brid.gy/docs#visibility it says

"Who can see me and my stuff?
Only the people who can already see you and your stuff, without bridging. Bridgy Fed only bridges fully public data, so if your account is private or protected or followers-only, it won't (can't!) bridge your account at all. Same with DMs and private/followers-only posts; it ignores those."

**Comment by snarfed** [comment]
AP => Bluesky posts aren't bridged historically, only going forward. https://fed.brid.gy/docs#posts-forward

**Comment by lucajet** [comment]
Hi Ryan, yes, i got it right then.
But i did posted and nothing passed yet, i will wait and see...

By the way, weren't external PDS limited in number of users/posts ?
Did they raised those limits ?

**Reply by snarfed** [reply]
Yes! They raised Bridgy Fed's limit specifically. Hopefully we'll be good citizens and they'll be happy with that decision! 😁
